maetio/expo-template
# Added storybook with Profile picture selector example

### 📋 **Initial Requirements**
*Please ensure that you have checked your PR for the following requirements:*  
1. ✅ **Compilation**: You have run `yarn tsc`, checking for no typescript errors. 
2. ✅ **Linting**: You have run `yarn lint` and fixed all linting and formatting errors.
3. ✅ **Testing**: You have rigorously tested your own code.
4. ✅ **Documentation**: If needed, the documentation is updated with any relevant information or links to new resources.  
5. ✅ **Developer Values**: Code is *modular*, *well-communicated*, *responsive*, and as *lightweight* as possible.


### 🔄 **What was changed?**
___  

> Added storybook as well as made an example of how to use it. The example given was being able to change the user profile picture using expo image picker.

![Screenshot 2022-11-18 144132](https://user-images.githubusercontent.com/95386742/202789046-a84447eb-758b-4768-a554-dde1aec83f45.png)

### 🤔 **Why were these changes made?**
___  

> Added these features since I believe adding storybook to Maet's stack is a great choice in expediting the time it can take making a component for the main app.


### ⚙️ **How to test:**
___ 

1.  run 'yarn'
2. run 'yarn storybook'. This should make the desktop UI
3. run 'yarn start' this should start the mobile UI on expo. You might encounter this error, 'can not find variable name HTMLElement'. If you see this check the readme in the .storybook folder and it will have instructions on how to fix this.


### 🔗 **Useful resources/links**:
___  
- Many examples in the .storybook folder README, but here is the [Storybook website](https://storybook.js.org/)
